mongodb/mongodb.php

```diff
@@ -244,6 +251,8 @@
             /**
              * Returns an array of tags describing this server in a replica set
              * @link http://php.net/manual/en/mongodb-driver-server.gettags.php
+             * @throws InvalidArgumentException on argument parsing errors.
+             * @return array
```

Comment:
@b1rdex Thanks for your time.  I've updated description to expose more information about return value type. I guess, since this method returns array of key-value pairs, `string[]` type hint can lead to wrong expectations: one can expect "list" structure, while it's a "dictionary".